nautobot/docs/additional-features/jobs.md

```diff
@@ -304,16 +304,20 @@ device = ObjectVar(
 )
 ```
 
-Multiple values can be specified by assigning a list to the dictionary key. It is also possible to reference the value of other fields in the form by prepending a dollar sign (`$`) to the variable's name.
+Multiple values can be specified by assigning a list to the dictionary key. It is also possible to reference the value of other fields in the form by prepending a dollar sign (`$`) to the variable's name. You can even traverse the model relationships this way (note the single underscore in the example - `tenant_region_id`).
```

Comment:
Not exactly - `tenant_group_id` is an [explicitly defined parameter on the FilterSet](https://github.com/nautobot/nautobot/blob/develop/nautobot/tenancy/filters.py#L71) (in this case, the `TenancyFilterSet` mixin that `SiteFilterSet` inherits from, so it works; `tenant_region_id` is not a defined FilterSet parameter so it won't work.

Reply:
Alright I reworded this a little and changed the `tenant_region_id` to be what I mean to: `tenant_group_id`.